lib/app/data/post_client.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:black_tax_and_white_benefits/app/domain/post.dart';
+import 'package:dio/dio.dart';
+import 'package:retrofit/http.dart';
+import 'package:retrofit/retrofit.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'post_client.g.dart';
+
+@RestApi(baseUrl: 'https://blacktaxandwhitebenefits.com/wp-json/wp/v2/')
+abstract class PostClient {
+  factory PostClient(Dio dio, {String baseUrl}) = _PostClient;
+
+  @GET('/posts?_embed=true')
+  Future<List<Post>> getPosts(
+    @Query('per_page') int n,
+  );
+}
+
+// coverage:ignore-start
```

Comment:
Please create a ticket for this. I will show you how to test this

Reply:
[Ticket created](https://anirac-tech.atlassian.net/browse/WA-22?atlOrigin=eyJpIjoiZjliY2M4NjgyNDc1NDIyNTkzZmNjOGFjNTA1MGEwYTciLCJwIjoiaiJ9)